bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/VehicleHandler.java

```diff
@@ -0,0 +1,575 @@
+            commandOptions.add(co);
+            StateOption so = new StateOption(entry, null);
+            stateOptions.add(so);
+        });
+        if (commandOptions.isEmpty()) {
+            commandOptions.add(new CommandOption("Initilaze", null));
+            stateOptions.add(new StateOption("Initilaze", null));
+        }
+        mmcop.setCommandOptions(new ChannelUID(thing.getUID(), GROUP_IMAGE, "image-view"), commandOptions);
+        mmsop.setStateOptions(new ChannelUID(thing.getUID(), GROUP_IMAGE, "image-view"), stateOptions);
+    }
+
+    private String getImage(String key) {
+        if (accountHandler.get().getImageApiKey().equals(NOT_SET)) {
+            logger.debug("Image API key not set");
+            return EMPTY;
+        }
+        String imageId = EMPTY;
+        if (imageStorage.get().containsKey(EXT_IMG_RES + config.get().vin)) {
+            String resources = imageStorage.get().get(EXT_IMG_RES + config.get().vin);
+            JSONObject jo = new JSONObject(resources);
+            if (jo.has(key)) {
+                imageId = jo.getString(key);
+            }
+        } else {
+            getImageResources();
+            return EMPTY;
+        }
+
+        String url = IMAGE_BASE_URL + "/images/" + imageId;
+        Request req = hc.newRequest(url);
+        req.header("x-api-key", accountHandler.get().getImageApiKey());
+        req.header(HttpHeader.ACCEPT, "*/*");
+        ContentResponse cr;
+        try {
+            cr = req.send();
+            byte[] response = cr.getContent();
+            return Base64.getEncoder().encodeToString(response);
+        } catch (InterruptedException | TimeoutException | ExecutionException e) {
+            logger.warn("Get Image {} error {}", url, e.getMessage());
+        }
+        return EMPTY;
+    }
+
+    private void call(String url) {
+        String requestUrl = String.format(url, config.get().vin);
+        // Calculate endpoint for debugging
+        String[] endpoint = requestUrl.split("/");
+        String finalEndpoint = endpoint[endpoint.length - 1];
+        // debug prefix contains Thing label and call endpoint for propper debugging
+        String debugPrefix = this.getThing().getLabel() + Constants.COLON + finalEndpoint;
+
+        Request req = hc.newRequest(requestUrl);
+        req.header(HttpHeader.AUTHORIZATION, "Bearer " + accountHandler.get().getToken());
+        try {
+            ContentResponse cr = req.send();
+            logger.debug("{} Response {} {}", debugPrefix, cr.getStatus(), cr.getContentAsString());
+            if (cr.getStatus() == 200) {
+                distributeContent(cr.getContentAsString().trim());
+            }
+        } catch (InterruptedException | TimeoutException | ExecutionException e) {
+            logger.warn("{} Error getting data {}", debugPrefix, e.getMessage());
+            fallbackCall(requestUrl);
+        }
+    }
+
+    /**
+     * Fallback solution with Java11 classes
```

Comment:
There are several issues with jetty and I'm still clueless how to solve them

1. Sporadic failures - not only this particular MercedesMe binding - 95% success but sometimes
````
2022-07-31 19:18:08.900 [INFO ] [ecastsolar.ForecastSolarPlaneHandler] - ForecastSolar Home South-West Call https://api.forecast.solar/estimate/54.321/8.976/35/45/5.5 failed java.io.EOFException: HttpConnectionOverHTTP@b17f6e::DecryptedEndPoint@1f8b25e{l=/192.168.1.20:45960,r=api.forecast.solar/148.251.178.234:443,OPEN,fill=-,flush=-,to=59981/0}
````
2. Picture calls all failed in my OH3.3 environment - that's why I removed the `ProtocolHandler` because I found this issue which matched exactly to the problem description. After this fix problem disappeared
3. [Permanent failure reported by a user](https://community.openhab.org/t/mercedes-me-binding/136852/21) on HTTP calls for mileage. Not reproducible on my side